packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -355,7 +358,10 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
 		this.verifyNotClosed();
 		if (this.isAttached()) {
 			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-			this.services!.deltaConnection.submit(content, localOpMetadata);
+			this.services!.deltaConnection.submit(
+				makeHandlesSerializable(content, this.serializer, this.handle),
```

Comment:
I'm curious what your approach would be for migrating external DDSes over.  This appears to be a breaking behavior change, in which case they'd need to update in the same PR that bumps to this version.

Although as I write that, I wonder if it's not actually, since I believe `makeHandlesSerializable` is idempotent.

Reply:
makeHandlesSerializable is idempotent